### PR TITLE
Move TestCommands from from sandbox-on-x-test-lib to test-common

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**

### CODE REVIEW

File: test-common/BUILD.bazel [hunk 1]
```diff
@@ -44,47 +44,41 @@ alias(
...
-                "@maven//:ch_qos_logback_logback_core",
-                "@maven//:io_grpc_grpc_context",
-                "@maven//:org_scalatest_scalatest_compatible",
-                "@maven//:org_slf4j_slf4j_api",
-            ],
-        ),
-    ]
-    for target in testtool_lf_versions
-]
-
```
Comment (remyhaemmerle-da) [inline test-common/BUILD.bazel:87]: We factorize that as there are no dependencies on LF.

File: test-common/BUILD.bazel [hunk 2]
```diff
@@ -44,47 +44,41 @@ alias(
...
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        "//daml-lf/archive:daml_lf_archive_reader",
+        "//daml-lf/data",
+        "//ledger-api/grpc-definitions:ledger_api_proto_scala",
+        "//ledger-api/testing-utils",
+        "//ledger/ledger-api-common",
+        "//ledger/ledger-api-domain",
```
Comment (remyhaemmerle-da) [inline test-common/BUILD.bazel:71]: is this not annoying, if you want to use test-common in canton ? 
@mziolekda @nmarton-da @atriantafyllos-da

Comment (atriantafyllos-da) [reply test-common/BUILD.bazel:71]: Yes keeping the duplication of the file seems more suitable.

Comment (mziolekda) [reply test-common/BUILD.bazel:71]: that is annoying indeed but it should work. We don't rely on anything inside /ledger anymore, all the libraries therein have been cleaned up and moved to other directories.
This dependency will be a new "dirty dependency" of the test-common that no-one will need.

Another suggestion, could TestCommands be moved to ledger-api-client instead?

**Pull request closed without merging**